### PR TITLE
Modularize the wetty service and add events

### PR DESCRIPTION
**Pull request by nosemeocurrenada**
This should help solving #8 

And having the service in a function will help if you ever need to host more than one Wetty

### CODE REVIEW

**Comment by butlerx** [comment]
could you fix the linting errors please, most should be fixed by `yarn lint:fix` as they are just spacing

File: app.js [hunk 1]
```diff
@@ -1,12 +1,8 @@
-const express = require('express');
-const http = require('http');
-const https = require('https');
-const path = require('path');
-const server = require('socket.io');
-const pty = require('pty.js');
-const fs = require('fs');
```
Comment (butlerx) [inline app.js]: fs is still used in this file

**Comment by nosemeocurrenada** [comment]
All done. Sorry about that.

**Comment by butlerx** [comment]
thanks for the help, great work

**Pull request merged**